pkg/version/metrics.go

```diff
@@ -56,12 +56,12 @@ var (
 			Title: "Disk Usage",
 			Queries: []kotsv1beta1.MetricQuery{
 				{
-					Query:  `sum(node_filesystem_size_bytes{job=~"node-exporter|kubernetes-service-endpoints",fstype!="",instance!=""} - node_filesystem_avail_bytes{job=~"node-exporter|kubernetes-service-endpoints",fstype!="",instance!=""}) by (instance)`,
-					Legend: "Used: {{ instance }}",
+					Query:  `node_filesystem_size_bytes{fstype=~"ext4|xfs",instance!="",job=~"node-exporter|kubernetes-service-endpoints"} - node_filesystem_avail_bytes{fstype=~"ext4|xfs",instance!="",job=~"node-exporter|kubernetes-service-endpoints"}`,
```

Comment:
How about having these 4 on the list: BTRFS, ZFS, XFS, EXT4?

Another option is to have the exclude list based on this: https://github.com/replicatedhq/kURL/blob/738f82e486f8864b5dc85573db72a27dcbd3a7cc/addons/collectd/v5/collectd.conf#L24-L32

Reply:
Thanks, I added the two that were missing: btrfs and zfs